submissions/Roophee/JS - DOM/index.js

```diff
@@ -1,84 +1,12 @@
 "use strict";
 
-const clickBurger = document.querySelector(".burger");
-const clickContent = document.querySelector(".content");
-const clickHeader = document.querySelector(".header");
-const clickfooter = document.querySelector(".footer");
-
+const burger = document.querySelector(".burger");
+const content = document.querySelector(".content");
 const deBar = document.querySelector(".aside");
-
-
-// MOVE ASIDE & CONTENT
-
-function hideSideBar(e) { 
-    e.preventDefault();
-    deBar.classList.toggle('hide__sidebar');
-    clickContent.classList.toggle('move_content');
-    clickBurger.classList.toggle('burger__stop');
-    console.log(e.type)
-
-};
-
-clickBurger.addEventListener('click', hideSideBar);
-
-// Enter-key event show-hide aside
-
-clickBurger.addEventListener('keydown', function (e) {
-    e.preventDefault();
-    if (e.keyCode == "13") {
-        deBar.classList.toggle('hide__sidebar');
-        clickContent.classList.toggle('move_content');
-        clickBurger.classList.toggle('burger__stop');
-    }
-}
-);
-
-// CLOSE ASIDE WHEN MOUSE CLICK OUT
-
-function hideSideBarClickAway(e) {
-    e.preventDefault();
-    for (let i = 0; i < deBar.classList.length; i++) {
-        if (deBar.classList[i] === 'hide__sidebar') {
-            deBar.classList.toggle('hide__sidebar');
-            clickBurger.classList.toggle('burger__stop');
-            clickContent.classList.toggle('move_content');
-        };
-    };
-};
-
-clickContent.addEventListener('click', hideSideBarClickAway);
-clickHeader.addEventListener('click', hideSideBarClickAway);
-clickfooter.addEventListener('click', hideSideBarClickAway);
-
-// ADD CONTENT IN .content
-
-function addContentOnClick() {
-    clickContent.innerHTML = "";
-
-    let contentRaw = contentStorage[this.innerText.toLowerCase()];
-    let makeContent = `<div class="content__header">
-    <h1 name="top">
-    ${contentRaw.title.toUpperCase()}
-    </h1>
-</div>
-<div class="content__body">
-    <img src="${contentRaw.imgurl}" alt="${contentRaw.alt}" class="content__img">
-    <p>
-        <span class="content__text">
-            ${contentRaw.text}
-        </span>
-    </p>
-</div>`;
-    clickContent.innerHTML = makeContent;
-};
-
-const navLinks = document.querySelectorAll(".nav__link");
-
-navLinks.forEach(item => item.addEventListener('click', addContentOnClick));
-
+const ulElement = document.querySelector(".nav__list");
```

Comment:
Try to use more meaningful names for variables, who describe entities in the proper way.

Reply:
Done.